subprojects/osgi-test/osgi-test.gradle

```diff
@@ -30,10 +30,27 @@ dependencies {
 }
 
 test {
+    jvmArgumentProviders.add(
+        new RuntimeBundlesProvider(files: configurations.testRuntimeBundles.asFileTree)
```

Comment:
does calling `asFileTree` here cause this configuration to be eagerly resolved? If so, we can pass instead `configurations.testRuntimeBundles` to the `RuntimeBundlesProvider` and then resolve it inside the `asArguments` function.

Reply:
It seems to be resolved at execution time, according to [this build scan](https://scans.gradle.com/s/i323oiwo3zq64/performance/dependency-resolution?toggled=dependency-resolution-execution)